common/src/main/java/com/graphicsfuzz/common/util/UpgradeShadingLanguageVersion.java

```diff
@@ -46,14 +49,23 @@
 import java.io.PrintStream;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import net.sourceforge.argparse4j.ArgumentParsers;
+import net.sourceforge.argparse4j.impl.Arguments;
 import net.sourceforge.argparse4j.inf.ArgumentParser;
 import net.sourceforge.argparse4j.inf.ArgumentParserException;
 import net.sourceforge.argparse4j.inf.Namespace;
 
 public class UpgradeShadingLanguageVersion extends ScopeTrackingVisitor {
 
+  // Flag for disabling renames. Used in a bit mask for conversion options.
+  static final int NORENAME = 1;
+
+  // Conversion flags. Used as a bit mask.
+  private int flags;
+
```

Comment:
Have a boolean field `bool renameUserDefined`. Add the same to the constructor.

Reply:
done